benchexec/tools/cmaesfuzz.py

```diff
@@ -29,6 +29,16 @@ def executable(self, tool_locator):
         return tool_locator.find_executable("fuzzer")
 
     def cmdline(self, executable, options, task, rlimits):
+        # add a time limit if not given
+        # that is hopefully sufficient to write all tests
+        if "-t" not in options:
+            # at least three seconds + 1% of overall time
+            timeout = int(rlimits.cputime*0.99 - 5)
```

Comment:
Mismatch with comment?

Reply:
Maybe you would also like to implement this offset within your tool? I.e., just passing the actual time limit to the tool, and the tool itself then deduces when it should stop?

Might be easier to update that way and more convenient for users who use the tool outside of BenchExec. The tool could also make a better estimate, i.e., depending on the size of the test suite... Or update the calculation if later versions are faster/slower.

Reply:
Maybe, but I'd rather not (the tool's time limit should be interpreted for finding test-cases, not writing them out)

Reply:
As you prefer.

But please make the comment consistent with the code and confirm that it works in tests now.